src/main/java/ua/hillel/treshchun/lessons/lesson12HW15WorkingWithFiles/FileLoggerConfigurationLoader.java

```diff
@@ -0,0 +1,100 @@
+package ua.hillel.treshchun.lessons.lesson12HW15WorkingWithFiles;
+
+import java.io.*;
+
+public class FileLoggerConfigurationLoader implements LoggerConfigurationLoader {
+
+
+    public FileLoggerConfiguration load(String fileName) throws FileConfigurationException {
+        try (BufferedReader br = new BufferedReader(new InputStreamReader(new FileInputStream(fileName)))) {
+            String filePath = "";
+            LoggingLevel level = null;
+            int maxSize = 0;
+            String format = "";
+            String str;
+            StringBuilder strB = new StringBuilder();
+            try {
+                if ((str = br.readLine()) != null) {
+                    while (str.isBlank()) {
+                        if ((str = br.readLine()) == null) {
+                            throw new FileConfigurationException("File format issue");
+                        }
+                    }
+                    strB.append(str);
+                    if (strB.substring(0, 6).equalsIgnoreCase("file: ")) {
```

Comment:
I would prefere use srt.split("[ ]") instead of strB.substring(0, 6) as some config-words length can be different and word your`re searching won`t every time be at the indexes 0, 6      0, 7     0, 10 as you described below...